collect_app/src/test/java/org/odk/collect/android/utilities/DateTimeUtilsTest.java

```diff
@@ -83,11 +81,11 @@ public void getDateWithSkippedDaylightSavingGapIfExists_returnsCorrectDateAndTim
     @Test
     public void skipDaylightSavingGapIfExistsTest() {
         DateTimeZone originalDefaultTimeZone = DateTimeZone.getDefault();
-        DateTimeZone.setDefault(DateTimeZone.forTimeZone(TimeZone.getTimeZone("Africa/Nairobi")));
+        DateTimeZone.setDefault(DateTimeZone.forID("Europe/Warsaw"));
 
-        // 1 Jan 1960 at 00:00:00 clocks were turned forward to 00:15:00
-        LocalDateTime ldtOriginal = new LocalDateTime().withYear(1960).withMonthOfYear(1).withDayOfMonth(1).withHourOfDay(0).withMinuteOfHour(0).withSecondOfMinute(0).withMillisOfSecond(0);
-        LocalDateTime ldtExpected = new LocalDateTime().withYear(1960).withMonthOfYear(1).withDayOfMonth(1).withHourOfDay(0).withMinuteOfHour(15).withSecondOfMinute(0).withMillisOfSecond(0);
+        // 29 March 2020 at 02:00:00 clocks were turned forward to 03:00:00
```

Comment:
I updated this test because apparently, that gap was a mistake https://github.com/JodaOrg/joda-time/issues/555